tool/jenkins/jenkins_build.sh

```diff
@@ -60,9 +60,9 @@ sleep 15
 echo "---------- Install pip, and mock -----------"
 # cmake, make, make install, and make package
 cat <<EOF
-docker exec madlib bash -c 'apt-get update; apt-get install -y python-pip; pip install mock' | tee $workdir/logs/madlib_compile.log
+docker exec madlib bash -c 'apt-get update; apt-get install -y python-pip; pip install mock scipy==1.2.1' | tee $workdir/logs/madlib_compile.log
 EOF
-docker exec madlib bash -c 'apt-get update; apt-get install -y python-pip; pip install mock' | tee $workdir/logs/madlib_compile.log
+docker exec madlib bash -c 'apt-get update; apt-get install -y python-pip; pip install mock scipy==1.2.1' | tee $workdir/logs/madlib_compile.log
 
```

Comment:
I hadn't noticed there was still an import for `scipy` in the DBSCAN module.  That was used in an earlier version, but now we are using `np.linalg.norm` to compute distances instead.  I've removed the dependency from DBSCAN, but we should keep this change in jenkins since kmeans does import `scipy`--not sure how it was working before.

Thanks for this and the gpdb5 port--great to see it working on all platforms now!